text/tooling/0001-extract-jam-from-packit.md

```diff
@@ -0,0 +1,48 @@
+# Extract `jam` into Dedicated Repository
+
+## Proposal
+
+Currently, the source for the `jam` cli tool currently resides inside of the
+`cargo` package which itself is inside of the `packit` repository. All `jam`
+source code should be extracted from its current location and be relocated into
+its own repository.
+
+## Motivation
+
+Initially `jam` was placed inside of the `packit` repository because it was the
+dedicated packaging tool for buildpacks that are written in `packit`. However,
+its functions have expanded beyond just packaging `packit` buildpacks and now
+encompass a wide variety of functions such as:
+* Summarizing the contents of a buildpackage
+* Updating `builder.toml` files
+* Updating metabuildpack `buildpack.toml` dependencies
+* Updating implementation `buildpack.toml` dependencies
+
+All of these are generic functions that could work with any buildpack
+regardless of whether or not it is written using `packit`. Furthermore, with
```

Comment:
I'm not sure that this is true. This only really applies to the updating of `builder.toml` and meta-buildpack `buildpack.toml` dependencies. The other two rely upon the structure of the `metadata` section in the `buildpack.toml` which is specific to `packit`.

Reply:
Hmm that is fair enough I can strike that from the commonality.

Reply:
This should be resolved, please check.